src/appleseed/renderer/modeling/input/inputarray.h

```diff
@@ -53,7 +53,7 @@ namespace renderer
 // Input formats.
 //
 
-enum InputFormat
+enum class InputFormat
 {
     InputFormatFloat,
```

Comment:
If the enum is scoped, the `InputFormat` prefixes become redundant and should be removed.